Two patches added to change destination directory for custom customer attribute pet_image from default `pub/media/customer` to `pub/media/pet_image` to allow displaying this image on the frontend.
Patches: dogma-customer-pet-image-change-destination-module-customer.patch, dogma-customer-pet-image-change-destination-module-eav.patch

index bf5e03b..7f0585d 100644
--- a/Model/Attribute/Data/File.php
+++ b/Model/Attribute/Data/File.php
@@ -192,7 +192,12 @@ class File extends \Magento\Eav\Model\Attribute\Data\AbstractData
         $fileData = $value;
 
         if (is_string($value) && !empty($value)) {
-            $dir = $this->_directory->getAbsolutePath($this->getAttribute()->getEntityType()->getEntityTypeCode());
+            if ($this->getAttribute()->getAttributeCode() == \Creativestyle\CustomizationDogmaOrder\Setup\Patch\Data\AddCustomerPetAttributes::PET_IMAGE_ATTRIBUTE) {
+                $dir = $this->_directory->getAbsolutePath(\Creativestyle\CustomizationDogmaOrder\Setup\Patch\Data\AddCustomerPetAttributes::PET_IMAGE_ATTRIBUTE);
+            } else {
+                $dir = $this->_directory->getAbsolutePath($this->getAttribute()->getEntityType()->getEntityTypeCode());
+            }
+
             $stat = $this->_directory->getDriver()->stat($dir . $value);
             $fileData = [
                 'size' => $stat['size'],
